aiida_vasp/utils/aiida_utils.py

```diff
@@ -87,6 +89,71 @@ def builder_interface(calc_cls):
     return False
 
 
+def new_structure(old_structure):
```

Comment:
Consider naming these functions `clone_structure` etc instead, as `new_something` may be read to create an empty node.

Reply:
Agree. I will use copy.

Reply:
Fixed